packages/db-dashboard/package.json

```diff
@@ -23,29 +23,31 @@
     "lint": "standard | snazzy"
   },
   "dependencies": {
-    "@fastify/static": "^6.5.0"
+    "@fastify/static": "^6.5.0",
+    "@mui/material": "^5.10.11",
+    "@platformatic/db-ra-data-rest": "workspace:*",
+    "inflected": "^2.1.0",
+    "react-admin": "^4.4.4"
```

Comment:
because I'm importing from 'react-admin' in `ReactAdmin.jsx`:
```
... 
import { Admin, Resource, ListGuesser, EditGuesser, ShowGuesser } from 'react-admin'
...
```

Reply:
The jsx files are bundled before publish and not compiled in production.

Reply:
Ok, thanks for the clarification, so should I remove these dependencies or move them in devDependency?

Reply:
can you tell me if I should move the above dependencies inside devDepencencies or remove them permanently, please?

Reply:
move to devDependencies.